lesson_13/maps_ts/src/lesson13.ts

```diff
@@ -3,5 +3,11 @@
  * https://leetcode.com/problems/permutation-difference-between-two-strings
  */
 export function findPermutationDifference(s: string, t: string): number {
-  return 0;
-}
+    let result = 0;
+    const strLen: number = s.length;
+    for (let i = 0; i < strLen; i++) {
+        const ch: string = s.charAt(i);
+        result = result + Math.abs(s.indexOf(ch) - t.indexOf(ch));
```

Comment:
This works, but the time complexity will be `O(n^2)`. Surely you can do better!

Reply:
HashMap implementation ready for review